src/hb-script-lang-table.h

```diff
@@ -0,0 +1,253 @@
+  { "gn",       { HB_SCRIPT_LATIN } },
+  { "gu",       { HB_SCRIPT_GUJARATI } },
+  { "gv",       { HB_SCRIPT_LATIN } },
+  { "ha",       { HB_SCRIPT_LATIN } },
+  { "haw",      { HB_SCRIPT_LATIN } },
+  { "he",       { HB_SCRIPT_HEBREW } },
+  { "hi",       { HB_SCRIPT_DEVANAGARI } },
+  { "hne",      { HB_SCRIPT_DEVANAGARI } },
+  { "ho",       { HB_SCRIPT_LATIN } },
+  { "hr",       { HB_SCRIPT_LATIN } },
+  { "hsb",      { HB_SCRIPT_LATIN } },
+  { "ht",       { HB_SCRIPT_LATIN } },
+  { "hu",       { HB_SCRIPT_LATIN } },
+  { "hy",       { HB_SCRIPT_ARMENIAN } },
+  { "hz",       { HB_SCRIPT_LATIN } },
+  { "ia",       { HB_SCRIPT_LATIN } },
+  { "id",       { HB_SCRIPT_LATIN } },
+  { "ie",       { HB_SCRIPT_LATIN } },
+  { "ig",       { HB_SCRIPT_LATIN } },
+  { "ii",       { HB_SCRIPT_YI } },
+  { "ik",       { HB_SCRIPT_CYRILLIC } },
+  { "io",       { HB_SCRIPT_LATIN } },
+  { "is",       { HB_SCRIPT_LATIN } },
+  { "it",       { HB_SCRIPT_LATIN } },
+  { "iu",       { HB_SCRIPT_CANADIAN_SYLLABICS } },
+  { "ja",       { HB_SCRIPT_HAN, HB_SCRIPT_KATAKANA, HB_SCRIPT_HIRAGANA } },
+  { "jv",       { HB_SCRIPT_LATIN } },
+  { "ka",       { HB_SCRIPT_GEORGIAN } },
+  { "kaa",      { HB_SCRIPT_CYRILLIC } },
+  { "kab",      { HB_SCRIPT_LATIN } },
+  { "ki",       { HB_SCRIPT_LATIN } },
+  { "kj",       { HB_SCRIPT_LATIN } },
+  { "kk",       { HB_SCRIPT_CYRILLIC } },
+  { "kl",       { HB_SCRIPT_LATIN } },
+  { "km",       { HB_SCRIPT_KHMER } },
+  { "kn",       { HB_SCRIPT_KANNADA } },
+  { "ko",       { HB_SCRIPT_HANGUL } },
+  { "kok",      { HB_SCRIPT_DEVANAGARI } },
+  { "kr",       { HB_SCRIPT_LATIN } },
+  { "ks",       { HB_SCRIPT_ARABIC } },
+  { "ku-am",    { HB_SCRIPT_CYRILLIC } },
+  { "ku-iq",    { HB_SCRIPT_ARABIC } },
+  { "ku-ir",    { HB_SCRIPT_ARABIC } },
+  { "ku-tr",    { HB_SCRIPT_LATIN } },
+  { "kum",      { HB_SCRIPT_CYRILLIC } },
+  { "kv",       { HB_SCRIPT_CYRILLIC } },
+  { "kw",       { HB_SCRIPT_LATIN } },
+  { "kwm",      { HB_SCRIPT_LATIN } },
+  { "ky",       { HB_SCRIPT_CYRILLIC } },
+  { "la",       { HB_SCRIPT_LATIN } },
+  { "lah",      { HB_SCRIPT_ARABIC } },
+  { "lb",       { HB_SCRIPT_LATIN } },
+  { "lez",      { HB_SCRIPT_CYRILLIC } },
+  { "lg",       { HB_SCRIPT_LATIN } },
+  { "li",       { HB_SCRIPT_LATIN } },
+  { "ln",       { HB_SCRIPT_LATIN } },
+  { "lo",       { HB_SCRIPT_LAO } },
+  { "lt",       { HB_SCRIPT_LATIN } },
+  { "lv",       { HB_SCRIPT_LATIN } },
+  { "mai",      { HB_SCRIPT_DEVANAGARI } },
+  { "mg",       { HB_SCRIPT_LATIN } },
+  { "mh",       { HB_SCRIPT_LATIN } },
+  { "mi",       { HB_SCRIPT_LATIN } },
+  { "mk",       { HB_SCRIPT_CYRILLIC } },
+  { "ml",       { HB_SCRIPT_MALAYALAM } },
+  { "mn-cn",    { HB_SCRIPT_MONGOLIAN } },
+  { "mn-mn",    { HB_SCRIPT_CYRILLIC } },
+  { "mni",      { HB_SCRIPT_BENGALI } },
+  { "mo",       { HB_SCRIPT_CYRILLIC, HB_SCRIPT_LATIN } },
+  { "mr",       { HB_SCRIPT_DEVANAGARI } },
+  { "ms",       { HB_SCRIPT_LATIN } },
```

Comment:
Malay is often written in Arabic script (AKA [Jawi](https://en.wikipedia.org/wiki/Jawi_script), but I’m not sure what language tag is used for this.